views/index.erb

```diff
@@ -1,9 +1,13 @@
 <form action="/" accept-charset="UTF-8" method="POST" class="w-50">
-    <label for="searchtext" class="form-label">Station Search <em>(use quotes for multi-word, e.g. shilshole "puget sound")</em></label>
-
+    <label for="searchtext" class="form-label">Station Search <em>(use double quotes for multi-word, e.g. shilshole "puget sound")</em></label>
```

Comment:
I disallowed single quotes because it broke for cases like `"hudson's bay"` where the first double quotes would be matched with the apostrophe of `hudson's`. I would also find it ambiguous if maybe you were searching for stations in locations that both had apostrophes like `hudson's mary's` (not sure if that case would ever come up and would probably be rarer).

IMO only using double quotes (and explicitly saying so) leads to a better user experience in the end even though it's more restrictive.

In case you still feel otherwise, what I'll do is I'll submit a commit that allows single or double quote tokenization (no mismatching) and allow you to choose what you want when you merge the code in.

Reply:
what could be cool is actually if in my thingy that I'll try out is if I identify whether or not a single quote is an apostrophe or a quote. (it'd probably be a dumb heuristic that'd break on plural words like friends')

Reply:
You know what, I like your explanation.  I'm convinced.  Let's stick with what you did.

Reply:
oki doki